TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robotSubSystems/drivetrain/Drivetrain.java

```diff
@@ -18,12 +18,13 @@ public class Drivetrain {
     private static Pose2d pose;
     public static Vector lastPosition;
     // equal to the last Autonomous position?
-    public static float lastTime = 0;
-    public static Vector lastVelocity = GlobalData.inAutonomous == true ? getVelocity_FieldCS() : null;
+    public static Vector lastVelocity = GlobalData.inAutonomous ? getVelocity_FieldCS() : null;
```

Comment:
Why is this var depends on auto or not?

Reply:
we already discussed that'it is because last velocity id calculated using road runner function, therefore it created the bug when we used to just disconnect from the rc